minisom.py

```diff
@@ -379,25 +390,40 @@ def train(self, data, num_iteration, random_order=False, verbose=False):
             Data matrix.
 
         num_iteration : int
-            Maximum number of iterations (one iteration per sample).
+            If use_epochs is True:
+                Number of epochs the SOM will be trained for
+            If use_epochs is False:
+                Maximum number of iterations (one iteration per sample).
+
         random_order : bool (default=False)
             If True, samples are picked in random order.
             Otherwise the samples are picked sequentially.
 
         verbose : bool (default=False)
             If True the status of the training
             will be printed at each iteration.
```

Comment:
We should specify here that an iteration is considered as weights update.

Reply:
Ok, but where exactly do you mean? Inside the explanation of "verbose"?

Reply:
yes, because the progress bar just shows a progressive index. In case that `use_epoch == False` the index goes from 0 to `max_iter`, otherwise it goes from 0 to `max_iter * number_of_samples`. This can be confusing for the user.

Reply:
ok, i changed it according to your suggestion.